test/shell/test_coverage_scalatest_resources.sh

```diff
@@ -5,7 +5,9 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 test_coverage_succeeds_resource_call() {
-    bazel coverage //test/coverage_scalatest_resources/consumer:tests
+    bazel coverage \
+          --instrumentation_filter=^//test/coverage_scalatest_resources[:/] \
```

Comment:
I guess this is an example why `--instrumentation_filter` was not checked in the previous implementation, but I think let's have it as it is consistent with how Bazel coverage works